packages/settings/src/pharmacies.ts

```diff
@@ -0,0 +1,27 @@
+const ALTO_ID = 'phr_01G9CM93X1NFP1C9H9K50DPKHX';
```

Comment:
should we make this `pharmacies.photon.ts`?

Reply:
The IDs are the same in all environments